server/mediaserver.go

```diff
@@ -116,14 +119,7 @@ func NewLivepeerServer(rtmpAddr string, httpAddr string, lpNode *core.LivepeerNo
 
 //StartServer starts the LPMS server
 func (s *LivepeerServer) StartMediaServer(ctx context.Context, transcodingOptions string) error {
-	bProfiles := make([]ffmpeg.VideoProfile, 0)
-	for _, opt := range strings.Split(transcodingOptions, ",") {
-		p, ok := ffmpeg.VideoProfileLookup[strings.TrimSpace(opt)]
-		if ok {
-			bProfiles = append(bProfiles, p)
-		}
-	}
-	BroadcastJobVideoProfiles = bProfiles
+	BroadcastJobVideoProfiles = parsePresets(strings.Split(transcodingOptions, ","))
```

Comment:
When would we opt to use a global variable, as we do here, as opposed to putting `BroadcastJobVideoProfiles` within the LivepeerServer` or `LivepeerNode` itself? I know we add `BroadcastJobVideoProfiles` to each `BroadcastSession`, and the value doesn't change, so perhaps that value doesn't need to live in a struct?

Reply:
Yeah, it's a node-wide default. Globals aren't great, putting it in a struct is another option [1] but this is how it's been done for a while, and it's a (mostly) read-only value.

The behavior of `BroadcastJobVideoProfiles` isn't really being changed here, only choosing whether we actually use that default or the set of options returned via webhook.

[1] This is why folks sometimes say that "singleton structs" stuffed with random values are no different than globals, eg LivepeerServer or LivepeerNode.

Reply:
> [1] This is why folks sometimes say that "singleton structs" stuffed with random values are no different than globals, eg LivepeerServer or LivepeerNode.

 Good point. I can see why that'd be the case. That's what I was wondering! 

> The behavior of `BroadcastJobVideoProfiles` isn't really being changed here, only choosing whether we actually use that default or the set of options returned via webhook.

Makes sense. Wouldn't really make a difference if it were included in `LivepeerNode` or `LivepeerServer`. The value is only read from a couple of times.